examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -24,8 +26,18 @@
 from collections import namedtuple
 
 from pickle import load
-from typing import List, Union
-Handler = Union[nonbonded.AM1CCCHandler, nonbonded.LennardJonesHandler] # TODO: relax this assumption
+from typing import List, Union, Tuple
+
+from typing import Union, Optional, Iterable, Any, Tuple, Dict
```

Comment:
Some duplicate imports here from above

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/360/commits/e359676e356dfd7d201425f8a08b56654b3769c7